tee-worker/ts-tests/package.json

```diff
@@ -2,7 +2,8 @@
 	"license": "ISC",
 	"scripts": {
 		"load-meta": "curl -s -H \"Content-Type: application/json\" -d '{\"id\":\"1\", \"jsonrpc\":\"2.0\", \"method\": \"state_getMetadata\", \"params\":[]}' http://localhost:9944 > litentry-metadata.json",
-		"generate": "yarn generate:defs && yarn generate:meta",
+		"format": "yarn prettier --write .",
+		"generate": "yarn generate:defs && yarn generate:meta && yarn prettier --write .",
```

Comment:
We can't say `&& yarn format` here? :thinking:

Reply:
well spot, updated now.